src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      action="#"
+    >
+      <input
+        class="form small"
+        type="text"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Keep your attributes correctly formatted. Since the `<input>` element has more than two attributes, each attribute should start on a new line with a 2-space indentation relative to the tag.